### PR TITLE
[DE] Add game modifiers `Easy`, `Fade Out` and `Fade In`

### PR DESCRIPTION
**Pull request by The-Last-Cookie**
## Notes

- `judgement` --> `Beurteilung`

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Gameplay/Game_modifier/Easy/de.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+In [osu!](/wiki/Game_mode/osu!) verringert die Mod Easy die [Circle-Size (CS)](/wiki/Client/Beatmap_editor/Song_Setup#circle-size), die [Approach-Rate (AR)](/wiki/Beatmapping/Approach_rate), die [allgemeine Schwierigkeit (OD)](/wiki/Client/Beatmap_editor/Song_Setup#overall-difficulty) und die [HP-Drain](/wiki/Client/Beatmap_editor/Song_Setup#hp-drain-rate) um die Hälfte.
+
+![EZ Gameplay in osu!](img/EZ-comparison-osu.jpg "Vergleich in osu! zwischen einem regulären Spiel (links) und einem Spiel, bei dem die Mod Easy aktiviert ist (rechts)")
+
+Allerdings ist es erwähnenswert, dass viele Spieler die Verwendung der Mod Easy nicht als hilfreich empfinden, um den relativen Schwierigkeitsgrad von Beatmaps zu verringern, insbesondere welche mit hoher Intensität. Das Argument hierfür lautet, dass die verringerte Approach-Rate eine unübersichtliche, schwer zu lesende Situation schafft, bei der die tatsächliche Geschwindigkeit der Beatmap visuell nicht sehr gut widergespiegelt wird (oben abgebildet).
+
+*Anmerkung: Die oben genannten Punkte werden in der Feature-Anfrage "[Let's talk Easy Mod](https://osu.ppy.sh/community/forums/topics/56606)" näher ausgeführt, um die Mod zu verbessern.*
```
Comment (FamousPig) [inline wiki/Gameplay/Game_modifier/Easy/de.md:47]:
```suggestion
*Anmerkung: Die oben genannten Punkte werden in der Feature-Anfrage "[Let's talk Easy Mod](https://osu.ppy.sh/community/forums/topics/56606)", um die Mod zu verbessern, näher ausgeführt.*
```

File: wiki/Gameplay/Game_modifier/Easy/de.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+In [osu!catch](/wiki/Game_mode/osu!catch) sind die Effekte die gleichen wie in osu!, mit ein paar Parallelen und Anpassungen: alle Fruits fallen langsamer (erhöhte AR), die Kamera ist "herangezoomt" (erhöhte CS) und zwei zusätzliche "Leben" werden bereitgestellt.
+
+Obwohl nur eine Achse verwendet wird, können die Fruits durch ihre erhöhte Größe und die langsamere Fallrate miteinander verklumpen, wodurch die gleichen gemeinsamen Probleme in der Schwierigkeit wie in osu! entstehen.
+
+![EZ Gameplay catch](img/EZ-comparison-catch.jpg "Vergleich in osu!catch zwischen einem regulären Spiel (links) und einem Spiel, bei dem die Mod Easy aktiviert ist (rechts)")
+
+### osu!mania
+
+In [osu!mania](/wiki/Game_mode/osu!mania) sind die Effekte die gleichen wie in osu!taiko, außer, dass dem Spieler die zwei zusätzlichen Leben *tatsächlich* bereitgestellt werden und dass die Slider-Geschwindigkeit sowie die standardmäßige Tastenanzahl dasselbe sein werden.
```
Comment (FamousPig) [inline wiki/Gameplay/Game_modifier/Easy/de.md:67]:
```suggestion
In [osu!mania](/wiki/Game_mode/osu!mania) sind die Effekte die gleichen wie in osu!taiko, außer, dass dem Spieler die zwei zusätzlichen Leben *tatsächlich* bereitgestellt werden und dass die Slider-Geschwindigkeit sowie die standardmäßige Tastenanzahl dieselbe sein werden.
```

File: wiki/Gameplay/Game_modifier/Fade_In/de.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+- Abkürzung: FI
+- Typ: Erhöhung der Schwierigkeit
+- Score-Multiplikator: 1,00x
+- Standard-Tastenkürzel: `F`
+- Kompatible Spielmodi: ![][osu!mania]
+- Variante: [Hidden](/wiki/Game_modifier/Hidden)
+
+## Beschreibung
+
+Die Mod **Fade In** ist eine [Spielmodifikation](/wiki/Game_modifier) für [osu!mania](/wiki/Game_mode/osu!mania), die die Schwierigkeit von [Beatmaps](/wiki/Beatmap) erhöht, indem die Notes verschwinden und dann beim Erreichen der [Beurteilungs](/wiki/Gameplay/Judgement)leiste erscheinen. Daher der Name.
```
Comment (FamousPig) [inline wiki/Gameplay/Game_modifier/Fade_In/de.md:30]:
```suggestion
Die Mod **Fade In** ist eine [Spielmodifikation](/wiki/Game_modifier) für [osu!mania](/wiki/Game_mode/osu!mania), die die Schwierigkeit von [Beatmaps](/wiki/Beatmap) erhöht, indem die Notes verschwinden und dann beim Erreichen der [Beurteilungsleiste](/wiki/Gameplay/Judgement) erscheinen. Daher der Name.
```

Not a big fan of turning half a word into a link.

**Review by FamousPig — CHANGES_REQUESTED** [review]

File: wiki/Gameplay/Game_modifier/Easy/de.md (hunk 2 above)
Comment (The-Last-Cookie) [reply wiki/Gameplay/Game_modifier/Easy/de.md:67]: I wasn't sure which was correct here really. Now that you mention it, is `dieselben` correct, due to the plural?

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**